include/quicly/cc.h

```diff
@@ -0,0 +1,38 @@
+/* Interface definition for quicly's congestion controller.
+ */
```

Comment:
I think you might want a copyright notice like the one you have in cc-reno.c here.